config/locales/en.yml

```diff
@@ -120,6 +120,7 @@ en:
     label: Activity
 
   articles:
+    article_development: Article Development
```

Comment:
We should keep the same UI as before, so "(article development)" instead of "Article Development", and the same for history.